apps/judicial-system/backend/src/app/modules/case/filters/case.filters.ts

```diff
@@ -61,38 +61,32 @@ function isStateHiddenFromRole(
   return getBlockedStates(role, institutionType).includes(state)
 }
 
-function getBlockedTypes(
+function getAllowedTypes(
   role: UserRole,
   forUpdate: boolean,
   institutionType?: InstitutionType,
 ): CaseType[] {
-  const blockedTypes: CaseType[] = []
-
-  if (role !== UserRole.STAFF) {
-    return blockedTypes
+  if (
+    [
+      UserRole.ADMIN,
```

Comment:
Admins should not have access to any cases. They should only handle user management. The reason they were not explicitly blocked before was that they are prevented from accessing cases through guards.

Reply:
Are you saying that historically we needed Admins to have access to cases and that has been addressed? So now we should remove admin here and return an empty list of allowed case types?

Reply:
No, admins have never had access, but they were blocked in guards not here. Now we are reversing the logic to give access rather than to block and should not grant any access to ADMINs.